init_db.sql

```diff
@@ -0,0 +1,134 @@
+-- MySQL dump 10.13  Distrib 8.0.24, for Win64 (x86_64)
```

Comment:
Hi. Can you tell me how you made such a file? If it's not too much trouble.

Reply:
I can, contact me.